precompile/allow_list.go

```diff
@@ -48,10 +48,10 @@ type AllowListConfig struct {
 // the addresses in [AllowListAdmins].
 func (c *AllowListConfig) Configure(state StateDB, precompileAddr common.Address) error {
 	for _, enabledAddr := range c.EnabledAddresses {
-		setAllowListRole(state, precompileAddr, enabledAddr, AllowListEnabled)
+		SetAllowListRole(state, precompileAddr, enabledAddr, AllowListEnabled)
```

Comment:
Should the allow list be its own package as well that can be imported to the other precompiles that use it?

Reply:
i left it out since it was not an actual precompile, but yeah we can move it.